specification/e-referrals-service-api.yaml

```diff
@@ -2392,30 +2392,57 @@ paths:
           $ref: 'components/schemas/responses/TooManyRequests.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
-  /R4/HealthcareService/{id}/_history/{version}:
-    get:
+  /STU3/Appointment/{id}:
+    put:
       security:
         - bearerAuth: []
       description: |
         ## Overview
         To be added....
-      
-      summary: A034 - Retrieve service version
-      operationId: a034-retrieve-service-version
+
+      summary: A034 - Update Appointment
+      operationId: a034-update-appointment
       parameters:
         - $ref: 'components/schemas/headers/request/BearerAuthorization.yaml'
-        - $ref: 'components/schemas/path/ServiceId.yaml'
-        - $ref: 'components/schemas/path/Version.yaml'
         - $ref: 'components/schemas/headers/request/OdsCode.yaml'
         - $ref: 'components/schemas/headers/request/BusinessFunction.yaml'
         - $ref: 'components/schemas/headers/request/CorrelationID.yaml'
+        - $ref: 'components/schemas/path/BookingId.yaml'
+        - $ref: 'components/schemas/headers/request/IfMatch.yaml'
+      requestBody:
+        required: true
+        content:
+          application/fhir+json:
+            schema:
+              $ref: 'components/schemas/eRS-Appointment.yaml'
+            examples:
+              minimal-booking:
+                summary: Minimum booking update For Cancellation
+                value:
+                  $ref: 'components/examples/updateAppointment/requests/MinimalBookingUpdateForCancel.json'
       responses:
+        '200':
+          $ref: 'components/schemas/responses/updateAppointment/200Response.yaml'
         '403':
           $ref: 'components/schemas/responses/Forbidden.yaml'
         '404':
-          $ref: 'components/schemas/responses/NotFoundVersioned.yaml'
-        '200':
-          $ref: 'components/schemas/responses/getService/200Response.yaml'
+          $ref: 'components/schemas/responses/NotFound.yaml'
+        '409':
+          $ref: 'components/schemas/responses/Conflict.yaml'
+        '415':
+          $ref: 'components/schemas/responses/UnsupportedMediaType.yaml'
+        '422':
+          $ref: 'components/schemas/responses/updateAppointment/422Response.yaml'
+        '423':
+          $ref: 'components/schemas/responses/Locked.yaml'
+        '429':
+          $ref: 'components/schemas/responses/TooManyRequests.yaml'
+        '500':
+          $ref: 'components/schemas/responses/InternalServerError.yaml'
+        '502':
+          $ref: 'components/schemas/responses/updateAppointment/502Response.yaml'
+        '504':
+          $ref: 'components/schemas/responses/updateAppointment/504Response.yaml'
```

Comment:
I think we need to include the 412 (PRECONTION_FAILED) response here also should the client not include an if-match header.